app/models/user.rb

```diff
@@ -78,8 +78,8 @@ def active_for_authentication?
     super && active
   end
 
-  def need_two_factor_authentication?(request)
-    MFA_ENABLED
+  def need_two_factor_authentication?(request = nil)
```

Comment:
Do you think this should be a class method instead? It doesn't seem to be directly related to a instance of the class

Reply:
I wanted to have like so but the gem `two_factor_authentication` requires this to be a instance method.